project/Build.scala

```diff
@@ -459,4 +460,22 @@ object Build extends sbt.Build {
     settings = commonSettings
   ) dependsOn(streaming % "test->test;compile->compile")
 
+  lazy val hbase = Project(
+    id = "gearpump-experiments-hbase",
+    base = file("experiments/hbase"),
+    settings = commonSettings ++
+      Seq(
+        resolvers ++= Seq(
+          "cloudera" at "https://repository.cloudera.com/artifactory/cloudera-repos"
+        )
+      ) ++
+      Seq(
+        libraryDependencies ++= Seq(
+          "org.apache.hadoop" % "hadoop-common" % hadoopVersion,
```

Comment:
Sure, I'm trying to test the HBaseSink with HBaseTestingUtility, but met some dependency problem, will fix that later.